core/src/main/java/com/dtp/core/notify/AbstractDtpNotifier.java

```diff
@@ -34,9 +34,7 @@
 
 import static com.dtp.common.constant.DynamicTpConst.TRACE_ID;
 import static com.dtp.common.constant.DynamicTpConst.UNKNOWN;
-import static com.dtp.common.constant.LarkNotifyConst.LARK_AT_FORMAT_OPENID;
-import static com.dtp.common.constant.LarkNotifyConst.LARK_AT_FORMAT_USERNAME;
-import static com.dtp.common.constant.LarkNotifyConst.LARK_OPENID_PREFIX;
+import static com.dtp.common.constant.LarkNotifyConst.*;
```

Comment:
You can adjust the settings of the idea, try not to use * to import, otherwise checkstyle will fail, you can compile it locally

Reply:
Please review the latest submission to confirm whether we still need to remove the executorWrapper.